contracts/hub/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "steak-hub"
-version = "2.0.0"
+version = "2.1.1"
```

Comment:
let's release this under v2.0.0

the previous PR you contributed (for Terra2) was tagged v2.0.0-rc0. we can tag this as rc1